topdown/topdown.go

```diff
@@ -538,11 +556,43 @@ func evalRefRuleCompleteDoc(ctx *Context, ref ast.Ref, path ast.Ref, rule *ast.R
 	})
 }
 
+func evalRefRuleCompleteDocFull(ctx *Context, ref ast.Ref, rules []*ast.Rule, iter Iterator) error {
+
+	var result ast.Value
+
+	for _, rule := range rules {
+
+		bindings := storage.NewBindings()
+		child := ctx.Child(rule, bindings)
+		isTrue := false
+
+		err := Eval(child, func(child *Context) error {
+			isTrue = true
+			return nil
```

Comment:
I've refactored this code path a bit to remove the redundancy. Also, this let me get rid of the unreachable/panic branches in the evalRefRuleResult function. Thanks!